jane.rb

```diff
@@ -69,13 +69,14 @@ def render_button(button)
 
   def render_device(device_name, buttons)
     html_device =
-    "<div class=\"col-md-4 col-sm-6\"><h3>#{device_name}</h3><div class=\"well\">\n"
+    # "<div class=\"col-md-4 col-sm-6\"><h3>#{device_name}</h3><div class=\"well\">\n"
+    "<div class=\"col-md-4 col-sm-6\"><div class=\"panel panel-default\"><div class=\"panel-heading\" style=\"font-size:1.5em; font-weight:bold\">#{device_name}</div><div class=\"panel-body\">\n"
```

Comment:
Lars mentioned that we should not use the style attribute but rather write a custom.css where we define and overwirte bootstrap styles